hikari/api/rest.py

```diff
@@ -7081,6 +7030,10 @@ async def edit_application_command(
         name: undefined.UndefinedOr[str] = undefined.UNDEFINED,
         description: undefined.UndefinedOr[str] = undefined.UNDEFINED,
         options: undefined.UndefinedOr[typing.Sequence[commands.CommandOption]] = undefined.UNDEFINED,
+        default_member_permissions: typing.Union[
```

Comment:
unless you special case this so that Permissions.NONE is special cased to be treated as null here then you're going to have to also allow None to be passed here